### PR TITLE
Add playwright

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by martinRenou** [comment]
Trying to revive adding playwright to conda-forge. Previous attempt: #14387

I patched playwright so that it does not ship the `node` executable. It also does not ship browsers.

File: recipes/playwright/0001-Remove-node.patch [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+diff --git a/playwright.cmd b/playwright.cmd
+new file mode 100644
+index 0000000..ffd94bb
+--- /dev/null
++++ b/playwright.cmd
+@@ -0,0 +1,3 @@
++@ECHO OFF
++SETLOCAL
++"%~dp0\%PREFIX%\bin\node.exe" "%~dp0\package\lib\cli\cli.js" %*
```
Comment (martinRenou) [inline recipes/playwright/0001-Remove-node.patch:22]: Not sure about this `%PREFIX%\bin\node.exe`?

**Comment by martinRenou** [comment]
@bollwyvl you might be interested by this?

**Comment by martinRenou** [comment]
Also if you'd like to be added as a maintainer for this recipe, please tell me!

**Comment by bollwyvl** [comment]
> Like pyppeteer does.

...yeah, i don't like that much either, and it took _years_ for them to not download it in a broken-by-default manner, which we had to patch around. It's been discussed over [on here](https://github.com/conda-forge/pyppeteer-feedstock/issues/3) whether we could do better w/r/t to provisioning the special-magic-ancient chromium it is tested with... but again, see speical-magic-ancient and therefore not-really-worth-my-time.

i _guess_, as a non-noarch package, as this must be, having multi-outputs that overload where it caches its downloads, and making those downloads packages could have legs... provided it's not _entirely_ random what version you'd be getting.

> added as a maintainer for this recipe,

nah, i'm good, but good luck!

**Pull request closed without merging**